dll/win32/shdocvw/CNSCBand.h

```diff
@@ -0,0 +1,245 @@
+/*
+ * PROJECT:     ReactOS shdocvw
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     NameSpace Control Band
+ * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#pragma once
+
+#define NSCBANDCLASSNAME L"ReactOS NameSpace Control Band"
```

Comment:
Should the class name match Windows in case somebody does FindWindowEx?

Reply:
Yes, it should. However, I can't understand the best implementation yet.